core-lib/src/generation.rs

```diff
@@ -119,8 +120,9 @@ impl LLMVMCore {
         let model = parameters
             .model
             .ok_or(CoreError::MissingParameter("model"))?;
-        let model_description =
-            ModelDescription::from_str(&model).map_err(|_| CoreError::ModelDescriptionParse)?;
+        let model_description = ModelDescription::from_str(&model)
+            .map_err(|_| CoreError::ModelDescriptionParse)
+            .unwrap();
```

Comment:
Is there a reason why we unwrap here? Or is this just a left over from debugging maybe?

Reply:
Yes, I had issues with the parser ignoring any error, so I had to check that for debugging.